packages/wallet/src/redux/protocols/direct-funding/actions.ts

```diff
@@ -1,17 +1,20 @@
 import * as actions from '../../actions';
 import { ActionConstructor } from '../../utils';
+import { TwoPartyPlayerIndex } from '../../types';
```

Comment:
Why does the direct-funding protocol need to change for this PR?

Reply:
The main change to the DirectFunding reducer was adding a exchangePostFundSetups` flag that can be set to determine whether the DirectFunding reducer sends Post Fund Setups after funding.

When topping up an existing ledger channel we don't want to send post fund setups for the ledger channel, we just want to deposit into the channel and then exchange updates after. However before these changes the direct funding reducer would automatically send postFundSetups for the ledger channel after deposit.

I added the `exchangePostFundSetup` flag so we can use the `DirectFunding` reducer from the `ledger-topup` protocol without breaking existing protocols.